app/build.gradle.kts

```diff
@@ -45,15 +46,16 @@ android {
 
     buildTypes {
         release {
-            isMinifyEnabled = true
-            isShrinkResources = true
-            proguardFiles(
-                getDefaultProguardFile("proguard-android-optimize.txt"),
-                "proguard-rules.pro"
-            )
+            //isMinifyEnabled = true
+            //isShrinkResources = true
```

Comment:
This optimizes **the APK** and thus takes extra time to run. We do not require these at the moment since the relative app performance is of no concern to the CI